tests/coverage/BUILD

```diff
@@ -8,14 +8,14 @@ scala_test(
     ],
     scala_toolchain_name = "test_zinc_2_13",
     runtime_deps = [
-        "@hamcrest_core//jar",
-        "@junit_interface//jar",
+        "@annex_test//:com_novocode_junit_interface",
```

Comment:
Nice does this mean you removed the last vestiges of the jvm imported deps and everything is now using rules_jvm_external? If so, that's awesome.

Reply:
It does!